pycbc/dq.py

```diff
@@ -164,13 +164,9 @@ def query_flag(ifo, segment_name, start_time, end_time,
                 flag_segments = data['segments']
 
         except Exception as e:
-            print(e)
             if source != 'any':
                 raise ValueError("Unable to find {} segments in GWOSC, check "
                                  "flag name or times".format(segment_name))
-            else:
```

Comment:
Ok, guess this  ended up a bit more annoying than I intended. I'm not sure I have a suggestion besides removing as you've done.

Reply:
I did consider turning the messages from `print()` to `logging.warn()` (so at least they stand out less), but then I figured a warning is unnecessary if things are going to work anyway. I can put the messages back if they are useful though.